packages/cli/src/cmds/agentInstaller/agentInstallerProcedure.ts

```diff
@@ -62,7 +62,7 @@ export default class AgentInstallerProcedure extends AgentProcedure {
 
     await this.installer.installAgent();
 
-    this.verifyProject();
+    await this.verifyProject();
```

Comment:
Good catch!

Reply:
This problem was hard to see on macOS, but really obvious on Windows. Running two instances of `mvn` in parallel caused errors when one tried to use a file that the other was downloading.